README.asciidoc

```diff
@@ -82,8 +82,13 @@ with https://github.com/andreyorst/plug.kak[plug.kak]. Add this code to your `ka
 
 ----
 plug "ul/kak-lsp" do %{
-    cargo build --release
-    cargo install
+    cargo build --release --locked
+    cargo install --force
+} config %{
+    hook global WinSetOption filetype=rust %{
+        lsp-start
+        lsp-enable
```

Comment:
It's better to call `lsp-enable-window` in WinSetOption hook.

Reply:
ok